website/docs/r/container_group.html.markdown

```diff
@@ -32,6 +32,10 @@ resource "azurerm_container_group" "aci-helloworld" {
         cpu ="0.5"
         memory =  "1.5"
         port = "80"
+        env_vars {
+            "env"="staging"

```

Comment:
it might sound odd, but in this current example the Environment Variable looks very similar to Tags (which we normally use `"Environment" = "Staging"` as an example) - as such would it make more sense to make this `"NODE_ENV" = "Staging"`?

Reply:
I made them the same intentionally, but this change is fine.